modules/kubernetes/services/dask-gateway/outputs.tf

```diff
@@ -7,10 +7,8 @@ output "config" {
   description = "Dask gateway /etc/dask/dask-gateway.yaml configuration"
   value = {
     gateway = {
-      address          = "http://web-public-${var.name}"
-      "proxy-address"  = "scheduler-public-${var.name}:8786"
```

Comment:
@brl0 is there a reason that `proxy-address` was not included in new changes?

Reply:
@costrouc, The scheduler proxy no longer runs as a separate pod and is handled by the traefik proxy now, and the `proxy-address` is inferred from `address`.